lib/elixir/lib/system.ex

```diff
@@ -1,16 +1,16 @@
 defmodule System do
   @moduledoc """
-  The System module provides functions that interact directly
+  The `System` module provides functions that interact directly
   with the VM or the host system.
 
   ## Time
 
-  The System module also provides functions that work with time.
-  Returning different times kept by the system as well as support
-  for different time units.
+  The `System` module also provides functions that work with time,
+  returning different times kept by the system with support for
+  different time units.
 
-  One of the complexities in relying on system time is that they
-  may be adjust. For example, when you enter and leave daylight
+  One of the complexities in relying on system times is that they
+  may be adjusted. For example, when you enter and leave daylight
```

Comment:
@josevalim did you mean to say "system time" or "system times" here? I believe it would be "system times" because we're referring to both system time (as in Erlang VM) and OS time, but I'm not sure :)